model_hub/examples/huggingface/token-classification/ner_trial.py

```diff
@@ -0,0 +1,117 @@
+import functools
+import logging
+
+import ner_utils
+import torch
+import transformers
+from attrdict import AttrDict
+from model_hub import huggingface as hf
+from model_hub import utils
+
+from determined.pytorch import DataLoader, LRScheduler, PyTorchTrialContext
+
+
+class NERTrial(hf.BaseTransformerTrial):
+    def __init__(self, context: PyTorchTrialContext) -> None:
+        utils.configure_logging(
```

Comment:
It seems like a reasonable number of this code is replicated in the default __init__ of BaseTransformerTrial -- is there an issue here preventing you from calling `super.__init__()`?

Reply:
yes, unfortunately we can't use `super.__init__()` currently due to a limitation of the datasets object.  see the comment above  `ner_utils.build_tokenized_datasets`.

Reply:
What is this limitation?

Reply:
I've bumped to a more recent version of HF datasets that fixed a serialization issue that they had.  With the new version, we can use super init.

Reply:
resolved by bumping to a more recent version of huggingface datasets that does not have this serialization issue.